packages/ia-icons/src/ia-icons.js

```diff
@@ -0,0 +1,130 @@
+import { LitElement } from 'lit-element';
+import advance from './icons/advance';
+import audio from './icons/audio';
+import close from './icons/close';
+import donate from './icons/donate';
+import ellipses from './icons/ellipses';
+import iaLogo from './icons/ia-logo';
+import images from './icons/images';
+import search from './icons/search';
+import software from './icons/software';
+import texts from './icons/texts';
+import upload from './icons/upload';
+import user from './icons/user';
+import video from './icons/video';
+import web from './icons/web';
+
+class Icon extends LitElement {
+  static get properties() {
+    return {
+      fill: { type: String },
+      stroke: { type: String },
```

Comment:
Can these be customized as CSS variables instead of properties?

Reply:
You know, when I first implemented these in ia-topnav, I couldn't use CSS variables because they were simply lit-html templates, but now that they're full-on encapsulated components I should be able to go that route. Let me give it a try.

Reply:
After looking at this for a bit, I can't figure out how we'd allow for a different fill and stroke color for each usage of the icon without passing them in as props. If there's multiple instances of the magnifying glass icon, for example, and each are a different fill color, they'd need a scoped CSS declaration for each that would set the fill color variable. I was trying to mimic the method of adding fill and stroke colors that SVG uses instead.

This might be something that the group should decide on before I go further with it.

Reply:
Does it work if you use more specific selectors like this?
```
<style>
#mag1 {
  --fillColor: blue;
}
#mag2 {
  --fillColor: green;
}
</style>

<ia-icon icon="magifying" id="#mag1"></ia-icon>
<ia-icon icon="magifying" id="#mag2"></ia-icon>
```

Reply:
Updated to use CSS variables. I wasn't thinking about how the cascade would allow re-declaration of the same CSS variables within a tighter scope, allowing each icon to have a different color. Also added an example of having an icon component with a default color.